tests/test_logme.py

```diff
@@ -0,0 +1,56 @@
+"""
+Test functions for the LogME metric (src/locomoset/metrics/logme.py)
+"""
+
+import numpy as np
+import pytest
+
+from locomoset.metrics.logme import logme
+
+
+def test_logme_n_samples_catch(dummy_features_perfect, dummy_labels):
+    """Test that the LogME metric catches that the sample size isn't great enough to
+    compute a consistent score
+    """
+    with pytest.raises(AssertionError):
+        logme(dummy_features_perfect, dummy_labels)
+
+
+def test_logme_perfect_features(dummy_features_perfect, dummy_labels):
+    """Test that LogME returns a value greater than zero for perfect features, with less
+    than zero only seen for random features"""
+    features = np.concatenate(
+        [
+            np.asarray(dummy_features_perfect)
+            for _ in range((3500 // dummy_features_perfect.shape[0] + 1))
```

Comment:
Could you add comments explaining the purpose of `3500 //` here and in other tests, and make a variable that's used across all the tests instead of having hardcoded 3500s?

Reply:
Have added a variable for it in `conftest.py` and added comments.